src/app/components/InlineSpan/index.jsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { C_SHADOW } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { getBodyCopy } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const InlineSpan = styled.span`
+  ${props => (props.script ? getBodyCopy(props.script) : '')};
+  color: ${C_SHADOW};
+  ${({ service }) => getSansRegular(service)}
```

Comment:
Does this need this? Would it not be better to inherit the font styles from the parent?

Reply:
Actually bodyCopy, color are inherited do maybe?

Reply:
I've removed these lines from the PR in e09938 and can't see any visual difference so I guess not needed.